mongo/integration/mtest/options.go

```diff
@@ -48,28 +48,37 @@ var (
 	falseBool = false
 )
 
+// ServerParameters specifies the parameters that must be set on the test server to run the test.
+type ServerParameters struct {
```

Comment:
It's unclear to me whether server parameters besides the two below will be referenced by future tests in the 1.1 schema. If that's the case, this struct is not sufficient and should probably be a map between strings and bools. That would require some cleverness around unmarshalling, and may warrant a small refactor of how I'm getting parameters from the server currently.

Reply:
As discussed offline, I've removed the `ServerParameters` type and am using a `map[string]bool` now. This design is agnostic to the names of params passed in by tests, which is probably better. The "getParameter" command now also happens in test `Setup()` and is cached in the `testContext` for use by `verifyServerParametersConstraints()` later. No change in unmarshalling was required.